selectel/mks.go

```diff
@@ -238,3 +239,27 @@ func compareTwoKubeVersionsByPatch(a, b string) (string, error) {
 
 	return b, nil
 }
+
+func mksNodegroupV1ParseID(id string) (string, string, error) {
+	parts := strings.Split(id, "/")
+	if len(parts) < 2 {
```

Comment:
I guess it's better to test if len(parts) != 2 since >2 parts is invalid either.